hypothesis-python/src/hypothesis/strategies/_internal/core.py

```diff
@@ -1677,15 +1687,24 @@ def complex_numbers(
             f"Cannot have allow_nan={allow_nan!r}, min_magnitude={min_magnitude!r} "
             f"max_magnitude={max_magnitude!r}"
         )
-
     check_type(bool, allow_subnormal, "allow_subnormal")
+    if width not in (64, 128):
```

Comment:
I'm inclined to support `width=32` here, given we support `width=16` for `st.floats()`.

Reply:
Yeah, why not. Just because [numpy does not support it](https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.complex64), doesn't mean we can't!